apps/cyberstorm-storybook/stories/components/Switch.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { Switch } from "@thunderstore/cyberstorm";
+import React, { useState } from "react";
+
+const meta = {
+  title: "Cyberstorm/Components/Switch",
+  component: Switch,
+} as ComponentMeta<typeof Switch>;
+
+const defaultArgs = {
```

Comment:
Defining "defaultArgs" like these, that used only in one place , seem counterproductive. Instead of reducing repetition, it adds boilerplate.